source/YTEPs/YTEP-0034.rst

```diff
@@ -0,0 +1,190 @@
+++++++++++++++++
+
+The name of each field shall be stored in both the ``EXTNAME`` and ``BTYPE``
+properties of the header. The units of the field shall be stored in the
+``BUNIT`` property of the header. 
+
+Unit Information
+++++++++++++++++
+
+Each header will be entirely self-describing as to the unit system of the
+dataset, including the dimensions of length, time, mass, velocity, 
+temperature, and magnetic field units. In most cases, these units will be
+derived from the underlying dataset used by ``FITSImageData`` to produce
+the file, but it will be possible for the user to specify their own code
+unit definitions in the instantiation of the ``FITSImageData`` object. For
+images created by subclasses of ``FITSImageData`` such as ``FITSSlice``, 
+``FITSProjection``, etc., the ``length_unit`` of the file will be given 
+by the units specified in the ``width`` keyword argument or be chosen 
+automatically based on the size of the image. 
+
+Coordinate System
++++++++++++++++++
+
+Each header shall have the coordinate system of the dataset stored in the 
+WCS keywords. These will set up a linear coordinate system with an origin
+in pixel space at the center of the image. The relevant keyword arguments
+are: 
+
+* ``CTYPE[123]``: The coordinate system type, all of which shall be 
+  ``"LINEAR"`` for all axes.
+* ``CUNIT[123]``: The units of the coordinate axes, in dimensions of 
+  length and in the specified ``length_unit`` of the FITS image. The units 
+  should be the same for all axes.
+* ``CRPIX[123]``: The reference pixel coordinate of the image, which
+  shall always be the center of the image: ``0.5*(n[xyz]+1)``, where
+  ``n[xyz]`` is the number of pixels in each dimension.
+* ``CDELT[123]``: The width of each pixel along each axis in the units 
+  specified by ``CUNIT[123]``.
+* ``CRVAL[123]``: The reference physical coordinate of the image, which 
+  corresponds to the same location as ``CRPIX[123]``.
+
+Other Metadata
+++++++++++++++
+
+Each header shall have the current time of the dataset stored in the header
+keyword ``"TIME"``, where the units shall be the code time units of the
+dataset. 
+
+Future iterations of this standard may allow for other optional metadata
+such as the redshift, etc., which can be checked upon instantiation of the
+``YTFITSDataset`` instance. 
+
+Example Header
+++++++++++++++
+
+The following is an example header for a density field created from a slice
+of a FLASH dataset, as printed out by the AstroPy command-line tool 
+``fitsheader``:
+
+.. code-block:: text
+
+    SIMPLE  =                    T / conforms to FITS standard
+    BITPIX  =                  -64 / array data type
+    NAXIS   =                    2 / number of array dimensions
+    NAXIS1  =                  512
+    NAXIS2  =                  512
+    EXTEND  =                    T
+    EXTNAME = 'DENSITY '           / extension name
+    BTYPE   = 'density '
+    BUNIT   = 'g/cm**3 '
+    LUNIT   =                  1.0 / kpc
```

Comment:
The FITS 4.0 standard says:

> If the units of the keyword value are specified in the comment of the header keyword, it is recommended that the units string be enclosed in square brackets (i.e., enclosed by ‘[’ and ‘]’) at the beginning of the comment field, separated from the slash (’/’) comment field delimiter by a single space character. An example, using a non-standard keyword, is `EXPTIME = 1200. / [s] exposure time in seconds`

So I would recommend putting the units in these comments in `[]`.

Reply:
@Cadair thanks for this suggestion--I have implemented it in the code and I'll also do it here.